### DISCUSSION TITLE
Code button disappears when width reduces below 768px

### DISCUSSION THREAD

**Discussion by eulerbeat5** [community/community#10524] [Projects and Issues]
### Normal:
![image](https://user-images.githubusercontent.com/52531715/150939371-4c57a6f2-dfad-451f-b84e-cca8f2a1ce18.png)

---

### Abnormal: where is the `Code` dropdown?
<img width="577" alt="View license" src="https://user-images.githubusercontent.com/52531715/150939402-70bc7333-a94c-4290-97b1-8303617f4f4b.png">

**Comment by HyperSuperMetaCtrl** [comment]
I tried today I think they "fixed" it. Thank God!
